index.html

```diff
@@ -120,7 +120,7 @@ <h2>Profesional Art Printing Data</h2>
                       <li>Ruby</li>
                   </ul>
               </div>
-                <button type="button">See project</button>
```

Comment:
- [ ] I kindly suggest that your `See Project` button should link to the Live Demo of the project or to the GitHub repository.